src/Data/Nat/Properties.agda

```diff
@@ -1185,6 +1190,46 @@ m⊔n≤m+n m n with ⊔-sel m n
 ... | inj₁ m⊔n≡m rewrite m⊔n≡m = m≤m+n m n
 ... | inj₂ m⊔n≡n rewrite m⊔n≡n = m≤n+m n m
 
+------------------------------------------------------------------------
+-- Other properties of _⊔_ and _*_
+
+*-distribˡ-⊔ : _*_ DistributesOverˡ _⊔_
+*-distribˡ-⊔ m zero o = begin-equality
+  m * (zero ⊔ o)                                        ≡⟨⟩
```

Comment:
Cleaner and easier to read I think if you use `0` here and elsewhere in the equational reasoning rather `zero`.